hbase-client/src/main/java/org/apache/hadoop/hbase/client/HBaseAdmin.java

```diff
@@ -1931,7 +1942,13 @@ public void splitRegion(final byte[] regionName, final byte[] splitPoint) throws
     if (regionServerPair.getSecond() == null) {
       throw new NoServerForRegionException(Bytes.toStringBinary(regionName));
     }
-    splitRegionAsync(regionServerPair.getFirst(), splitPoint);
+    try {
+      splitRegionAsync(regionServerPair.getFirst(), splitPoint).get();
```

Comment:
`splitRegion` will return immediately without waiting for or surfacing any exception from the operation. This seems wrong. Yet, it is a semantic change over previously released versions. For discussion.

Reply:
Same, will ignore.